### ISSUE TITLE
CVodeGetRootInfo error

### ISSUE THREAD

**Issue by antoine-soubret** [JuliaLang/Sundials.jl#64]
I am trying to run the cvode_Roberts_dns.jl example in Julia 0.4.5.
However, I receive the following error as soon as CVodeGetRootInfo is called:
```
<ERROR: MethodError: `CVodeGetRootInfo` has no method matching CVodeGetRootInfo(::Ptr{Void}, ::Array{Int32,1})
Closest candidates are:
  CVodeGetRootInfo(::Ptr{Void}, ::Ptr{Int32})
 [inlined code] from none:5
 in anonymous at no file:0>
```
CVodeGetRootInfo is called with the following command:
```
  flagr = Sundials.CVodeGetRootInfo(cvode_mem, rootsfound)
```
and rootsfound  is initialized with
```
rootsfound = round(Int32,[0, 0])
```
So, I could not figure out what is the problem here.
My understanding is that Julia arrays are automatically converted into Ptr by Julia.

Thanks for your help,

Antoine

**Comment by acroy** [comment]
Have a look at #61, which fixes this problem (essentially, one has to use `pointer(rootsfound)` in this case). I was hoping that the ida example also get's fixed in the same PR, but maybe we should just merge that one...

**Comment by acroy** [comment]
Should be fixed via #65 - let us know if there are problems.
